### DISCUSSION TITLE
Pushbullet

### DISCUSSION THREAD

**Discussion by sloke123** [alexjustesen/speedtest-tracker#437] [Ideas]
First of all, Thank you very much for making such a beautiful app. 🙏🏻🙏🏻🙏🏻
Do you have any plans for Pushbullet notification?

**Comment by alexjustesen** [comment]
Added to the list https://github.com/alexjustesen/speedtest-tracker/issues/40

**Comment by sloke123** [comment]
Thank you very much.☺
